ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/424_model_serving_bias_metrics.robot

```diff
@@ -0,0 +1,151 @@
+    ...                 deployed prior to enabling the TrustyAI service
+    [Tags]    Smoke
+    ...       Tier1   ODS-2482    ODS-2479
+    Open Model Serving Home Page
+    Serve Model    project_name=${PRJ_TITLE}    model_name=${MODEL_ALPHA}    framework=${framework_onnx}    existing_data_connection=${TRUE}
+    ...    data_connection_name=model-serving-connection    model_path=${MODEL_PATH_ALPHA}    model_server=${RUNTIME_NAME}
+    ${runtime_name} =    Replace String Using Regexp    string=${RUNTIME_NAME}    pattern=\\s    replace_with=-
+    ${runtime_name} =    Convert To Lower Case    ${runtime_name}
+    Wait For Pods To Be Ready    label_selector=name=modelmesh-serving-${runtime_name}    namespace=${PRJ_TITLE}
+    Verify Model Status    ${MODEL_ALPHA}    success
+    Install And Verify TrustyAI Service
+    Wait Until Keyword Succeeds  5 min  10 sec  Verify Model Is Registered with TrustyAI Service
+    Send Batch Inference Data to Model     model_name=${MODEL_ALPHA}     project_name=${PRJ_TITLE}
+    ${modelId}=   Get ModelId For A Deployed Model    modelId=${MODEL_ALPHA}
+    ${modelId}    Replace String    ${modelId}    "    ${EMPTY}
+    Schedule Bias Metrics request via CLI     metricsType=dir   modelId=${modelId}    protectedAttribute="customer_data_input-3"
+    ...       favorableOutcome=0   outcomeName="predict"    privilegedAttribute=1.0    unprivilegedAttribute=0.0
+    Verify TrustyAI Metrics Exists In Observe Metrics    trustyai_dir    retry_attempts=2    user_type=basic
+    Remove TrustyAI Service
+
+*** Keywords ***
+Bias Metrics Suite Setup
+    [Documentation]    Setup to configure TrustyAI metrics
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+    Log    Enabling and Configuring User Workload Monitoring
+    Enable User Workload Monitoring
+    Configure User Workload Monitoring
+    Verify User Workload Monitoring Configuration
+    Launch Data Science Project Main Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    Create S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=model-serving-connection
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=${aws_bucket}
+    Create Model Server    token=${FALSE}    server_name=${RUNTIME_NAME}
+
+Bias Metrics Suite Teardown
+    [Documentation]     Bias Metrics Suite Teardown
+    Delete Data Science Project From CLI    displayed_name=${PRJ_TITLE}
+    RHOSi Teardown
+
+Verify User Workload Monitoring Configuration
+    [Documentation]    Verifies that the ALL monitoring components for user-defined-projects
+    ...    are READY in the ${DEFAULT_MONITORING_NS} namespace
+    Wait For Pods Status  namespace=${DEFAULT_MONITORING_NS}  timeout=60
+    Log  Verified Applications NS: ${DEFAULT_MONITORING_NS}  console=yes
```

Comment:
could you move this keyword in a common resource file? maybe the same where you have created the keyword to enable UWM